app/locales/en/messages.po

```diff
@@ -642,8 +642,8 @@ msgid "controls.filters.show-legend.toggle"
 msgstr "Show legend titles"
 
 #: app/configurator/components/filters.tsx
-msgid "controls.filters.show-legend.tooltip"
-msgstr "Allow users to change Legend visibility"
+msgid "controls.filters.show-legend-toggle"
+msgstr "Allow users to change legend visibility"
```

Comment:
@noahonyejese I think still this string should mention legend **title**?

<img width="302" alt="Screenshot 2025-02-11 at 11 35 45" src="https://github.com/user-attachments/assets/734bfd67-1035-4fc4-b2e3-72afdc255918" />